src/main/java/org/sqlite/jdbc3/JDBC3DatabaseMetaData.java

```diff
@@ -1245,12 +1248,13 @@ public ResultSet getPrimaryKeys(String c, String s, String table) throws SQLExce
      */
     public ResultSet getExportedKeys(String catalog, String schema, String table)
             throws SQLException {
-        PrimaryKeyFinder pkFinder = new PrimaryKeyFinder(table);
+        PrimaryKeyFinder pkFinder = new PrimaryKeyFinder(table, schema);
         String[] pkColumns = pkFinder.getColumns();
         Statement stat = conn.createStatement();
 
         catalog = (catalog != null) ? quote(catalog) : null;
-        schema = (schema != null) ? quote(schema) : null;
```

Comment:
Why it was removed?

Reply:
Was replaced by
```
        String quotedSchema = (schema != null) ? quote(schema) : null;
```
Because we don't want String quotes for identifiers which we use for 
```
                   stat.executeQuery("select name from " + getSchemaPrefix(schema) + "sqlite_master where type = "
                        + "'table'")) {
```

Reply:
I don't get it.

Before your change, it was assigned to a class field that might be used in various places, including by consumers of this library.

Was this change made because of this single call to `getSchemaPrefix`? Are you sure this is a good trade-off?

Reply:
Which is somewhat bad pattern in itself